app/main.py

```diff
@@ -1 +1,43 @@
-# write your code here
+from __future__ import annotations
+
+
+class Animal:
+    alive = []
+
+    def __init__(self,
+                 name: str,
+                 health: int = 100,
+                 hidden: bool = False) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def remove_dead(self) -> None:
+        Animal.alive.remove(self)
+
+    def deadly_actions(self, bite_power: int = 50) -> None:
```

Comment:
You should use verbs in methods naming to represent action. Please, change the method name